doc/releases/changelog-dev.md

```diff
@@ -185,7 +185,9 @@
 
 <h3>Breaking changes</h3>
 
-* The target wires of the unitary for `ControlledQubitUnitary` are no longer available via `op.hyperparameters["u_wires"]`.
+* `Operator.inv()` and the `Operator.inverse` setter are removed. Please use `qml.adjoint` or `qml.pow` instead.
+  [(#3618)](https://github.com/PennyLaneAI/pennylane/pull/3618)
+
   Instead, they can be accesses via `op.base.wires` or `op.target_wires`.
```

Comment:
I'm guessing this was deleted by accident
```suggestion

* The target wires of the unitary for `ControlledQubitUnitary` are no longer available via `op.hyperparameters["u_wires"]`.
  Instead, they can be accesses via `op.base.wires` or `op.target_wires`.
```

Reply:
oops.